Documentation/watch-recovery.md

```diff
@@ -0,0 +1,81 @@
+# Break system locally and watch recovery
+
+First, make sure you are running a sample app. Follow the [Tectonic Sandbox tutorials][first-app] to launch a simple app, or use one of your own. Once an app is running, break the cluster and watch how it recovers.
+
+One of the core features of Kubernetes is that it’s designed to maintain the desired state defined by operators and app admins. Now that we have an app running, let’s see what happen when we intentionally kill a copy of the API server, which is the brains of the cluster. The API server is run as a Deployment with multiple Pods, to ensure High Availability. Each one of these runs on a master node.
+
+## Simulate Pod failure
+
+First, let’s demonstrate that the Control Plane is run in High Availability by intentionally killing an API server Pod process, and watching it recover. This will also show the rescheduling process of Replication Controllers and reconciliation loops.
+
+Open a terminal and a browser window with Tectonic Console. Keep them open side by side for this exercise. Kubernetes will rebuild the killed containers quickly.
+
+First, use Tectonic Console to determine which nodes are running the API server.
+Go to *Workloads > Pods*, and enter ‘api’ in the search field.
+
+Then, select one of the nodes listed, and SSH to its IP address.
+
+```
+Sssh core@ip
+```
+
+Remember that the kubelet runs on **reconciliation loops** which are constantly work to keep the system in a stable state. When we execute the command to kill this API server, the kubelet will notice immediately.
```

Comment:
The grammar in the first sentence is weird.

Should it be:
* "...runs on **reconciliation loops** which are constantly work*ing...*" (work --> working) or 
* "...run on **reconciliation loop** which ~~are~~ constantly work" (remove "are" from the sentence)?